Tests/GraphQL/HyperwalletTransferMethodConfigurationTests.swift

```diff
@@ -183,6 +130,91 @@ class HyperwalletTransferMethodConfigurationTests: XCTestCase {
         XCTAssertEqual(branchIdMask?.scrubRegex, "\\s")
     }
 
+    func testRetrieveTransferMethodConfigurationFeeAndProcessingTime_success() {
```

Comment:
update the test name to whatever actual function name will be

Reply:
Renamed.